sdks/python/apache_beam/options/pipeline_options.py

```diff
@@ -1619,7 +1619,7 @@ def _add_argparse_args(cls, parser):
         help=(
             'Chooses which pickle library to use. Options are dill, '
             'cloudpickle or default.'),
-        choices=['cloudpickle', 'default', 'dill'])
+        choices=['cloudpickle', 'default', 'dill', 'dill_unsafe'])
```

Comment:
possible other ways to name this concept:

dill_any
dill_untested 
dill_any_version